src/statics/projectNav.ts

```diff
@@ -17,6 +25,10 @@ const PROJECT_NAV = [
 	{
 		TITLE: TEXT.PROJECT_NAV.RESOURCES,
 		REF: ROUTES.RESOURCES
+	},
+	{
+		TITLE: TEXT.PROJECT_NAV.PROJECTTIMELINE,
+		REF: ROUTES.PROJECTTIMELINE
```

Comment:
It may be good to add an underscore between the words :)

```suggestion
		TITLE: TEXT.PROJECT_NAV.PROJECT_TIMELINE,
		REF: ROUTES.PROJECT_TIMELINE
```